src/Lesson8/keyboard.java

```diff
@@ -0,0 +1,30 @@
+package Lesson8;
+
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class keyboard {
+    public static void main(String[] args) {
+        readData();
+
+    }
+
+    public static void readData() {
+        ArrayList list = new ArrayList();
+        try {
+            while (true) {
+                Scanner sc = new Scanner(System.in);
+                float number = sc.nextFloat();
+                list.add(number);
+            }
```

Comment:
Add any sout after scanner as it's not clear what user should do after running the code.